src/mixins/nr-affiliation-mixin.ts

```diff
@@ -91,32 +91,36 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
   }
 
   /** Creates temporary business record and returns business identifier. */
-  private async createBusiness (accountId: number, nr: NameRequestI): Promise<string> {
+  private async createBusiness (
+    accountId: number, nr: NameRequestI, isNewAffiliation: boolean
+  ): Promise<string> {
     const createBusinessResponse =
-      await BusinessServices.createBusiness(this.getBusinessRequest(accountId, nr))
+      await BusinessServices.createBusiness(this.getBusinessRequest(accountId, nr)).catch(() => null)
 
-    if (createBusinessResponse.status === CREATED) {
-      return createBusinessResponse.data?.filing?.business?.identifier as string
-    }
+    if (createBusinessResponse?.status !== CREATED) {
```

Comment:
Use conditional chaining in case response is null/undefined (eg, HTTP 500 response -- not something I originally tseted with!).